src/main/java/Demo.java

```diff
@@ -1,7 +1,100 @@
 package main.java;
 
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Demo {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        calculator1();
+    }
+
+    public static void calculator1() {
+        Scanner sc = new Scanner(System.in);
+        String inputString;
+        System.out.println("Enter a string :");
+        System.out.println("Trying to push directly to git.");
+        inputString = sc.nextLine();
+
+
+        String[] ary = inputString.split("");
+        System.out.println(Arrays.toString(ary));
+        int c = 1;
+        int a;
+        int b;
+
+
+        for (int i = 1; i < ary.length; i = i + 2) {
+            switch (ary[i]) {
+                case "", ".", "," -> {
+                    System.out.println(-1);
+                    return;
+                }
+
+                case "!" -> {
+                    System.out.println(Integer.parseInt(ary[i - 1]));
+                    a = Integer.parseInt(ary[i - 1]);
+                    System.out.println(a);
+                    while (a > 0) {
+                        c *= a;
+                        a--;
+                    }
+                }
+                case "*" -> {
+                    a = Integer.parseInt(ary[i - 1]);
+                    b = Integer.parseInt(ary[i + 1]);
+                    c = a * b;
+                }
+                case "/" -> {
+                    a = Integer.parseInt(ary[i - 1]);
+                    b = Integer.parseInt(ary[i + 1]);
+                    if (!(b == 0)){
+                    c = a / b;}
+                    else {
+                        System.out.println(-1);
+                    }
+                    
+                }
+                case "+" -> {
+                    a = Integer.parseInt(ary[i - 1]);
+                    b = Integer.parseInt(ary[i + 1]);
+                    c = a + b;
+                }
+                case "-" -> {
+                    a = Integer.parseInt(ary[i - 1]);
+                    b = Integer.parseInt(ary[i + 1]);
+                    c = a - b;
+                }
+
+            }
+            System.out.println(c);
+
+        }
     }
 }
+
+//    It's 1949 and you are tasked to lead programmer's team in order to implement a brand new calculator. The calculator is very simple though: it can add (+), multiple(x), divide(%) and take(-). It does not and will not support any brackets or memory. It will only ever support whole numbers.
```

Comment:
No need comments. They could be used initially, but should not end up in the published code